packages/expo-updates/ios/EXUpdates/Update/Update.swift

```diff
@@ -166,13 +176,12 @@ public class Update: NSObject {
         config: config,
         database: database
       )
-    } else {
-      return BareUpdate.update(
-        withBareManifest: BareManifest(rawManifestJSON: withEmbeddedManifest),
-        config: config,
-        database: database
-      )
     }
+    return BareUpdate.update(
```

Comment:
This change is just to fix SwiftLint linting, and is not related to the bug fixed in this PR.